kerchunk/grib2.py

```diff
@@ -582,3 +584,66 @@ def correct_hrrr_subhf_step(group: Dict) -> Dict:
     group["refs"]["step/0"] = enocded_val
 
     return group
+
+
+def parse_grib_idx(
+    basename: str,
+    suffix: str = "idx",
+    storage_options: Dict = {},
```

Comment:
```suggestion
    storage_options: Optional[Dict] = None,
```
It is considered bad practice to allow mutable default arguments. You will require `**storage_options` -> `**(storage_options or {})` below to conpensate.